src/tests/ftest/daos_test/daos_core_test.yaml

```diff
@@ -40,10 +40,6 @@ daos_tests:
       daos_test: r
       test_name: rebuild tests
       args: -s3 -u subtests="28"
-    test_r_29:
-      daos_test: r
-      test_name: rebuild tests
-      args: -s3 -u subtests="29"
     test_r_30:
```

Comment:
You said REBUILD30 in the email?  Is that actually 29?

Reply:
oh, -u "29" will run REBUILD30.  Sorry for confusing, I should change the REBUILD name.

Reply:
Ok, no problem.  Just checking if it was a typo